staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -643,7 +647,8 @@ func (c completedConfig) New(name string, delegationTarget DelegationTarget) (*G
 		readyzChecks:     c.ReadyzChecks,
 		livezGracePeriod: c.LivezGracePeriod,
 
-		DiscoveryGroupManager: discovery.NewRootAPIsHandler(c.DiscoveryAddresses, c.Serializer),
+		DiscoveryGroupManager:    discovery.NewRootAPIsHandler(c.DiscoveryAddresses, c.Serializer),
+		DiscoveryResourceManager: discoveryv1.NewResourceManager(c.Serializer),
```

Comment:
Probably not good naming since the naming of these fields suggests they are connected when they are in fact NOT related (other than both servicing discovery documents)

Reply:
Maybe something like AggregatedDiscoveryManager or AggregatedDiscoveryResourceManager?